CI/test_coverage.py

```diff
@@ -63,7 +63,7 @@ def call(cmd):
 if not os.path.exists(coverage_dir):
     os.makedirs(coverage_dir)
 
-excludes = ["-e", "../Tests/", "-e", ".*json\.hpp"]
+excludes = ["-e", "../Tests/", "-e", ".*json.hpp"]
```

Comment:
@paulgessinger I'm not that sure about this. Why did we use `\.` instead of a simple `.`? Can it be confused with something else I'm missing?

Reply:
I think, this was intended to be a regex forgetting, that python strings would treat them differently. With the raw string `r"\."` it should work fine. I would actually prefer to also have this a raw string for clarity.
```suggestion
excludes = ["-e", "../Tests/", "-e", r".*json\.hpp"]
```